src/forms/inputs/cloudinary-file-input/cloudinary-uploader.js

```diff
@@ -78,7 +78,7 @@ const DEFAULT_REQUEST_OPTIONS = {
 const FILE_NAME_PATTERN = /[\s?&#\\%<>]/gi
 
 // Throws an error when a required param is not found
-function requireParam(paramName, context = 'Error') {
+function requireParam(paramName, context) {
```

Comment:
This function is never used without passing `context`, so we don't need this default